addons/web/static/src/core/effects/rainbow_man.scss

```diff
@@ -1,256 +1,156 @@
-    background: center center / contain no-repeat;
-    animation: reward-float $reward-base-time ease-in-out $reward-base-time infinite alternate;
-    @include size(34%, 56.67%);
-  }
-
-  .o_reward_stars {
-    display: block;
-    @include size($reward-size * 0.75, $reward-size / 2);
-    @include media-breakpoint-down(sm) {
-      @include size($reward-size-mobile * 0.75, $reward-size-mobile / 2);
+    .o_reward_rainbow_man,
+    foreignObject {
+        // Could be replaced by the "overflow-visible" class after the migration to BS5
+        overflow: visible !important;
     }
-    @include o-position-absolute(18%, 7%);
-
-    svg {
-      transform-origin: center center;
-      @include o-position-absolute(28%, $left: 3%);
-      animation: reward-stars $reward-base-time ease-in-out 0s infinite alternate-reverse;
-
-      &.star2,
-      &.star4 {
-        animation: reward-stars ($reward-base-time * 1.2) ease-in-out 0s infinite alternate;
-      }
-
-      &.star2 {
-        left: 20%;
-        top: 2%;
-      }
-
-      &.star3 {
-        left: 49%;
-        top: 6%;
-      }
-
-      &.star4 {
-        left: 70%;
-        top: 27%;
-      }
+    .o_reward_rainbow_man {
+        max-width: 400px;
+        inset: 0;
     }
-  }
-
-  .o_reward_thumbup {
-    width: 40px;
-    display: block;
-    animation: reward-scale ($reward-base-time * 0.5) ease-in-out 0s infinite alternate;
-    @include o-position-absolute(63%, auto, auto, 65%);
-  }
-
-  .o_reward_msg_container {
-    will-change: transform;
-    padding-top: 11%;
-    width: 70%;
-    margin-left: 15%;
-
-    // Translate before animate
-    transform: translateY(5px);
-
-    animation: reward-float $reward-base-time ease-in-out $reward-base-time infinite
-      alternate-reverse;
-    @include o-position-absolute(85%, auto, auto, 0%);
-
-    .o_reward_thumb_right {
-      height: 40px;
-      z-index: 1;
-      @include o-position-absolute(0, auto, auto, 16%);
+    .o_reward_rainbow_line {
+        animation: reward-rainbow $reward-base-time * 0.8 ease-out 1 forwards;
     }
-
-    .o_reward_msg {
-      margin-left: 7%;
-      margin-top: -9.5%;
-      padding: 25px 15px 20px;
-      background: white;
-      border: 1px solid #ecf1ff;
-      border-top-width: 0;
-      display: inline-block;
-
-      // Reset margins for first and penultimate childs (the last one is shadow)
-      *:first-child {
-        margin-top: 0;
-      }
+    .o_reward_face_group {
+        animation: reward-jump $reward-base-time * 0.8 ease-in-out 1;
     }
-
-    .o_reward_msg_content {
-      position: relative;
-      font-family: sans-serif;
-      text-align: left;
-      color: $reward-text-color;
+    .o_reward_face_wrap {
+        animation: reward-rotate $reward-base-time * 0.8 cubic-bezier(.51,.92,.24,1.15) 1;
+    }
+    .o_reward_face {
+        animation: reward-float $reward-base-time ease-in-out $reward-base-time infinite alternate;
+    }
+    .o_reward_star_01,
```

Comment:
Related selectors, both can stay in the same line 
https://cssguidelin.es/#anatomy-of-a-ruleset